app/assets/stylesheets/helpers/_header.scss

```diff
@@ -88,10 +88,13 @@
       display: block;
       margin: 0;
       border: 0;
-      height: 35px;
+      min-height: 36px;
       padding: 6px 0;
-      @include ie-lte(7){
-        height: 23px;
+      @include ie-lte(8) {
```

Comment:
I changed it just because the dimensions were broken otherwise. But now I've dug a bit deeper into it. IE8 seems to have a problem with `box-sizing` together with `min-height`/`min-width` (and I guess `max-height`/`max-width` as well, but I haven't tested that). Unfortunately caniuse.com doesn't mention that, might be worth reporting. I found [evidence on SO](http://stackoverflow.com/questions/9508262/min-height-min-width-doesnt-respect-box-sizing-in-some-browsers). And I created [this reduced test case](https://dl.dropboxusercontent.com/u/4667354/gds/min-height-ie8.html) which demonstrates the issue reproducibly (if you open it in IE8).